public/modules/nav/navController.js

```diff
@@ -2,8 +2,25 @@
  * Created by brillwill on 16/9/14.
  */
 app.controller("navController", ["$scope", "$rootScope", function($scope, $rootScope){
+	$scope.hasLoggedIn = false;
+
     $scope.goLogin = function () {
         $rootScope.$state.go("nav.login");
-    }
+    };
+
+    $rootScope.$on('loggedIn',function(){
+    	$scope.hasLoggedIn = true;
```

Comment:
Duplicated. not need this variable.  You can check rootScope currentUser or token to see if any user logged in or not.